cmake/hpx.cmake

```diff
@@ -0,0 +1,45 @@
+#------------------------------------------------------------------------------#
+# Copyright (c) 2014 Los Alamos National Security, LLC
+# All rights reserved.
+#------------------------------------------------------------------------------#
+
+#------------------------------------------------------------------------------#
+# Add option to enable HPX
+#------------------------------------------------------------------------------#
+
+option(ENABLE_HPX "Enable HPX" OFF)
+
+if(ENABLE_HPX)
+
+#------------------------------------------------------------------------------#
+# Find HPX
+#------------------------------------------------------------------------------#
+
+  find_package(HPX REQUIRED NO_CMAKE_PACKAGE_REGISTRY)
+
+  if(NOT HPX_FOUND)
+      message(FATAL_ERROR "HPX is required
+                     for this build configuration")
+  endif(NOT HPX_FOUND)
```

Comment:
Isn’t that block unnecessary as there is a `REQUIRED` above?

Reply:
Fair enough. Will fix.